flumeserver/api/utils.go

```diff
@@ -273,7 +276,7 @@ func getBlocks(ctx context.Context, db *sql.DB, includeTxs bool, chainid uint64,
 	return results, nil
 }
 
-func getPendingTransactions(ctx context.Context, db *sql.DB, offset, limit int, chainid uint64, whereClause string, params ...interface{}) ([]map[string]interface{}, error) {
+func getPendingTransactions(ctx context.Context, db *sql.DB, offset *int, limit int, chainid uint64, whereClause string, params ...interface{}) ([]map[string]interface{}, error) {
```

Comment:
I'm curious why you made offset an *int instead of an `int`. That made a lot of other conversions necessary, when - at the times you're calling this function - you could do `getPendingTransactions(ctx, db, *offset, limit, chainid, whereclause, params...)`

You're never modifying offset, so I don't see that you need it to be a pointer.

Reply:
READ THIS FIRST